node/pkg/driver/device_connectivity/device_connectivity_fc.go

```diff
@@ -82,7 +82,7 @@ func (r OsDeviceConnectivityFc) GetMpathDevice(volumeId string, lunId int, array
 	}
 
 	for _, arrayIdentifier := range arrayIdentifiers {
-		wwn := "0x" + arrayIdentifier
+		wwn := "0x" + strings.ToLower(string(arrayIdentifier))
```

Comment:
you can use bytes.ToLower if arrayIdentifier is a byte slice:
`fmt.Sprintf("0x%s", bytes.ToLower(arrayIdentifier))`

Reply:
It is a string

Reply:
make sure you close this comment if there all was done